src/screens/IndexScreen.js

```diff
@@ -279,6 +270,13 @@ export const IndexScreen = ({ navigation, route }) => {
                 ListHeaderComponent={
                   showFilter ? (
                     <DropdownHeader {...{ query, queryVariables, data, updateListData }} />
+                  ) : subCategories?.length ? (
+                    <CategoryList
+                      navigation={navigation}
+                      data={subCategories}
+                      horizontal={false}
+                      noSectionHeader={true}
+                    />
```

Comment:
currently this depends on `showFilter`, which could result in a missing category list, ih a filter is shown. both needs to be possible at the same time.